### ISSUE TITLE
Style_loss is normalized by batch_size twice

### ISSUE THREAD

**Issue by Mona77** [xunhuang1995/AdaIN-style#30]
Hi,
Thanks for sharing your code publicly.
In your implementation style_loss is summation of **mean_square_error** of **mean** and **std**, this [line](https://github.com/xunhuang1995/AdaIN-style/blob/6afb2873c50353081d3636863b9a563769704ef0/lib/StyleINLossModule.lua#L88
).

`self.mean_loss = self.mean_criterion:forward(self.input_mean, self.target_mean)`

But may I ask why you have normalized this value by **batch size**?

`self.mean_loss = self.mean_loss / N -- normalized w.r.t. batch size`

Wouldn't MSE criteria take average over N and C?

**Comment by xunhuang1995** [comment]
Hi, the mean_criterion has sizeAverage set to False, so it's not normalized by batch size at the first place.
